src/constructs/aws/ServerSideWebsite.ts

```diff
@@ -222,6 +222,7 @@ export class ServerSideWebsite extends AwsConstruct {
         return {
             url: Fn.join("", ["https://", domain]),
             cname: this.distribution.distributionDomainName,
+            bucketName: this.bucket.bucketName,
```

Comment:
While bucketName is self explanatory in the scope of the storage construct, maybe within the scope of the server side website construct, something like assetsBucketName would help developer make more sense of this variable. WDYT ?

Reply:
Good idea! I will update the change